bikeshed/unsortedJunk.py

```diff
@@ -1671,3 +1672,64 @@ def forceCrossorigin(doc):
         ):
             continue
         el.set("crossorigin", "")
+
+
+def addImageSize(doc):
+    if doc.md.imgAutoSize is False:
+        return
+    imgElements = findAll("img", doc)
+    for el in imgElements:
+        if el.get("width") or el.get("height") or el.get("noautosize"):
+            continue
+        if el.get("noautosize") is not None:
+            el.attrib.pop("noautosize")
+            continue
+        src = el.get("src")
+        srcset = el.get("srcset")
+        res = 1
+        if src is None and srcset is None:
+            warn("<img> elements must have at least one of src or srcset.")
+            continue
+        elif src is not None and srcset is not None:
+            continue
+        elif src is None:
+            m = re.match(r"^[ \t\n]*([^ \t\n]+)[ \t\n]+(\d+)x[ \t\n]*$", srcset)
+            if m is None:
+                die(
+                    "Couldn't parse srcset attribute: {0}\nBikeshed only supports a single image followed by an integer resolution.",
```

Comment:
As I said in https://github.com/tabatkins/bikeshed/pull/2008#discussion_r605999896, this isn't valid HTML anyway since there's not src attribute. So either the author is targeting bikeshed, and they should use syntax bikeshed recognizes, or they're not, and they should add at least and src attribute (and maybe width and height). I'll add a suggestion along these lines.